### PR TITLE
Update information on Student Organization and Placement Test

### PR DESCRIPTION
**Pull request by Robert-Jia00129**

### CODE REVIEW

File: CSC.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## 目录
+
+## CSC 简介
+
+明尼苏达大学（UMN）的中国学生社区（CSC）致力于支持中国国际学生过渡到美国的生活。CSC 成立于 2023 年，它的理念是为 UMN 的中国学生量身定做一个强大的社区，分享宝贵的资源和促进文化理解与适应。
+
+## CSC 微信公众号
+
+> 公众号 ID：gh_970b17d03a2a
+> 公众号二维码：
```
Comment (TrotylTN) [inline CSC.md]: Should we link an image here?

File: CSC.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## 公众号精选
+- 福利｜CSC & Horizon Auto新学期接机活动来啦～ [链接](https://mp.weixin.qq.com/s/QjpOnNQKjVKv9FNc0DPEeg)
+- CSC福利｜新生办卡活动 [链接](https://mp.weixin.qq.com/s/ih-b6Jl7oAbMJ72wpXwNCA)
+
+
+## 版权与免责协议
```
Comment (TrotylTN) [inline CSC.md:60]: Please copy the copyright info from other documentation

File: CSC.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# 中国学生社区 (Chinese Student Community) 相关内容
```
Comment (TrotylTN) [inline CSC.md:29]: Please use https://raw.githubusercontent.com/mingcns/mingcns.github.io/master/appendix/new_doc_template.md as template for the correct doc gen behavior.

File: hcc.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Hospitality 相关内容
```
Comment (TrotylTN) [inline hcc.md]: Ditto, please use the template for the correct doc gen behavior

File: images/CSC/temp.md [hunk 5]
```diff
@@ -0,0 +1 @@
+asdf
```
Comment (TrotylTN) [inline images/CSC/temp.md]: Do we need this?

File: main_doc.md
Comment (TrotylTN) [inline main_doc.md:1]: Please split this to another PR for better clarity (it's not related to other changes in this PR)

**Review by TrotylTN — CHANGES_REQUESTED** [review]

File: CSC.md (hunk 1 above)
Comment (Robert-Jia00129) [reply CSC.md]: I'm waiting for the images to be updated so I could include a permanent link here. 

Are there any other ways of doing it?

Comment (TrotylTN) [reply CSC.md]: They could be done in the same PR, e.g. you could upload an image to sub dir of /images, then refer the images here like https://www.mingcns.org/images/xxx/yyy.png and it will work after the PR.

File: images/CSC/temp.md (hunk 5 above)
Comment (Robert-Jia00129) [reply images/CSC/temp.md]: Deleted. It was used just to create a new directory

File: main_doc.md
Comment (Robert-Jia00129) [reply main_doc.md:1]: This was already commited, are there any suggestions on how to do this? Shall I copy the original file and paste it in?

File: CSC.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
+---
+layout: page
+title: "中国学生社区 (Chinese Student Community)"
+permalink: /CSC/
```
Comment (TrotylTN) [inline CSC.md:4]: Don't use capital char in permalink, keept them consistent with the existing undercase_snake_format.

Comment (TrotylTN) [reply CSC.md:4]: also, for CSC and HCC images directory, it's also highly recommended to use undercase_snake_format

File: images/CSC/CSC.JPG
Comment (TrotylTN) [inline images/CSC/CSC.JPG:1]: maybe name the file as CSC_QRCode.jpg?

Don't forget to update the link referencing here.

File: images/HCC/IMG_1407.PNG
Comment (TrotylTN) [inline images/HCC/IMG_1407.PNG:1]: Please update the file name to reflect what the image means instead of the system ordinal file name.

File: main_doc.md [hunk 7]
```diff
@@ -111,7 +111,8 @@ _(仅限本科新生与本科转学生)_ [New Student Checklist](http://www.chec
 * 如果国际生新生 Orientation 只显示5到7月的日期，请发邮件联系对应学院的新生联络邮箱来选择8月的 Orientation 日期。在第一次做出选择后，Orientation 的日期后续 **可以** 在 Checklist 里更改。
 
 * Placement Test
-    * Placement test 中<font color="red">数学必考</font>，分配结果（课号）MATH1241/1271/1371 (Calculus I) 为该测试的最高水平。
+    * Placement test 中<font color="red">数学必考<sup>*</sup></font>，分配结果（课号）MATH1241/1271/1371 (Calculus I) 为该测试的最高水平。
+      * *(若有AP/IB/Alevel数学可免考数学，详见[https://admissions.tc.umn.edu/hs-counselor-guidebook/transferring-credit](https://admissions.tc.umn.edu/hs-counselor-guidebook/transferring-credit))
```
Comment (TrotylTN) [inline main_doc.md:115]: Nit: it's "A-level"

File: main_doc.md [hunk 8]
```diff
@@ -111,7 +111,8 @@ _(仅限本科新生与本科转学生)_ [New Student Checklist](http://www.chec
 * 如果国际生新生 Orientation 只显示5到7月的日期，请发邮件联系对应学院的新生联络邮箱来选择8月的 Orientation 日期。在第一次做出选择后，Orientation 的日期后续 **可以** 在 Checklist 里更改。
 
 * Placement Test
-    * Placement test 中<font color="red">数学必考</font>，分配结果（课号）MATH1241/1271/1371 (Calculus I) 为该测试的最高水平。
+    * Placement test 中<font color="red">数学必考<sup>*</sup></font>，分配结果（课号）MATH1241/1271/1371 (Calculus I) 为该测试的最高水平。
```
Comment (TrotylTN) [inline main_doc.md:114]: We don't have to use <sup>*</sup> and  * to annotate the following content

Markdown's built-in intent should be enough for the following details.

File: main_doc.md [hunk 9]
```diff
@@ -1198,7 +1199,15 @@ Admission Requirements 有时候又被称作 **Declaration Requirement** ，这
...
-3. **The Aurora Center**  
+3. **Gopher Chauffeur**
+这是Boynton Health为学生单独开设的开车接送服务，但是只有在夜间才有，而且也只覆盖小区周围一公里的距离。
+但是如果晚上需要从学校回家，直接在软件上订车就会有一辆印着学校Gopher的车来接你。
+软件下载[链接](http://onelink.to/5kmgpc)
+营业时间：
+周日-周三: 8:30 pm-12:30 am
+周四-周六: 10:00 pm-1:30 am
+
+5. **The Aurora Center**
```
Comment (TrotylTN) [inline main_doc.md]: Should this be 4 instead of 5?

**Review by TrotylTN — CHANGES_REQUESTED** [review]
Some nitpicky issues, generally LGTM

File: images/CSC/CSC.JPG
Comment (Robert-Jia00129) [reply images/CSC/CSC.JPG:1]: Thanks for reminding that.

**Review by TrotylTN — APPROVED** [review]
